### DISCUSSION TITLE
Is there support for non-public extensions without installing as a dev extension?

### DISCUSSION THREAD

**Discussion by cv4x** [zed-industries/zed#15632] [Feature Requests]
As an editor designed for entire teams to use together, is there any support for a team to develop a private extension (ie. to add a language server for their internal framework, proprietary stack, etc.) without having to publish it publicly in the central extension catalog?

Some extensions simply don't make sense to add to the central list inside the application, but the only alternative seems to be manual installation as a "dev" extension, which seems cumbersome in contrast to the auto-install/auto-update functionality offered by the settings file for officially offered extensions.

**Comment by bajrangCoder** [comment]
At this time , there is no such feature available in zed.

You will need to use the `dev extension` method.

**Comment by pintert3** [comment]
Will this ever be implemented?
